snapcast/control/server.py

```diff
@@ -272,6 +272,9 @@ def group(self, group_identifier):
 
     def stream(self, stream_identifier):
         """Get a stream."""
+        if stream_identifier not in self._streams:
+            self._synchronize_if_stream_missing(stream_identifier)
```

Comment:
Not sure if we should add it to stream() as well or let the calling application handle it. Behavior is kind of strange as it raises a keyerror and then updates. @happyleavesaoc what do you think?